server/package.json

```diff
@@ -368,10 +368,10 @@
     "terser-webpack-plugin": "^4.2.3",
```

Comment:
Can we remove this now? And also the `terser-webpack-plugin.d.ts` types file too?

Reply:
removed!

Reply:
oh dang, I discarded at some point to try a cleaner approach to something or the other, must have lost removing this when I did that. I've fixed it up and pushed that change up now. Thanks for catching this!